homeassistant/components/tado/device_tracker.py

```diff
@@ -55,6 +55,8 @@ async def async_get_scanner(
         translation_key = "import_aborted"
         if import_result.get("reason") == "import_failed":
             translation_key = "import_failed"
+        elif import_result.get("reason") == "import_failed_invalid_auth":
+            translation_key = "failed_import_invalid_auth"
```

Comment:
Can we choose one or the other?

Reply:
As in, naming wise, "import_failed_invalid_auth" and "failed_import_invalid_auth" are really similar

Reply:
One. Removed the elif.

Reply:
Oh I meant the names of the reason and the translation key

Reply:
I kept it in line with the strings and the reason descriptions:

![image](https://github.com/home-assistant/core/assets/5011203/71999657-74cb-44a6-9ec0-16e72668538b)

![image](https://github.com/home-assistant/core/assets/5011203/73dd30f5-4af1-4ac1-8a8e-c6637da02a07)